File: Password Factory/Controls/ZoomButton.m
//
//  ZoomButton.m
//  Password Factory
//
//  Created by Cristiana Yambo on 11/14/17.
//  Copyright © 2017 Cristiana Yambo. All rights reserved.
//

#import "ZoomButton.h"
#import "StyleKit.h"
@implementation ZoomButton

-(instancetype)initWithCoder:(NSCoder *)coder {
    self = [super initWithCoder:coder];
    self.bordered = NO;
    self.imagePosition = NSImageOnly;
    [self setButtonType:NSMomentaryChangeButton];
    self.bezelStyle = NSRegularSquareBezelStyle;
    self.image = [StyleKit imageOfZoom];
    [self setAlphaValue:0.25];
    return self;
    
}

@end

File: Password Factory iOS/Controls/ZoomButton.swift
//
//  ZoomButton.swift
//  Password Factory iOS
//
//  Created by Cristiana Yambo on 12/4/17.
//  Copyright © 2017 Cristiana Yambo. All rights reserved.
//

import UIKit
@IBDesignable

/// Zoom Button
class ZoomButton: UIButton {
    override func prepareForInterfaceBuilder() {
        super.prepareForInterfaceBuilder()
        setDisplay()
    }
    override func awakeFromNib() {
        setDisplay()
    }
    func setDisplay() {
        super.awakeFromNib()
        setBackgroundImage(StyleKit.imageOfZoom(), for: .normal)
        setTitle("", for: .normal)
    }

}
